includes/utilities.inc

```diff
@@ -80,6 +77,41 @@ function islandora_newspaper_get_newspaper($object) {
  *     - issued: A DateTime object repersenting the date the issue was released.
```

Comment:
Not your fault, was there from forever but could be good chance to fix "repersenting"